docs/messaging-service/sms-config.md

```diff
@@ -0,0 +1,28 @@
+# SMS gateway configuration
+
+Messaging service supports 2 ways of sending SMS messages out of the box:
+
+- By integrating with the [CM.com SMS gateway](https://www.cm.com/sms/)
+- By integrating with a custom API
+
+## CM.com integration configuration
```

Comment:
same as for email, is the word "integration" necessary?

Reply:
fixed